### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by UkrainianBanderasCat** [UkrainianBanderasCat/Digital-Logic-Sim2#42] [Ideas]
I collected here some ideas your and from itch:
-~~Copy and paste chips~~
-Display on chip
-~~Replace clock in chips to input pin "clk"~~
-~~Edit chips or just view content~~
-Fix game optimization
-Bus
-Multiple input without OR
-put I/O on top and bottom
-Output or input on BOTH sides with an option to select (I am working on it)

If I forget something say to me

**Comment by AlphaCodingPilot** [comment]
What do you mean by "Display on chip" and "Multiple input without OR"?

**Reply by UkrainianBanderasCat** [reply]
Display on chip it's if you created a chip with screen or 7SegmentDisplay it will on chip and Multiple input without OR it's when you have some chips and need their output connect to one input, now you can do it only by gate or

**Comment by wolfysta** [comment]
Having the ability to change the order in which chips appear on the bottom of the screen

**Comment by AlphaCodingPilot** [comment]
I'll work on copy and pasting chips next.

**Reply by AlphaCodingPilot** [reply]
I added it #48. Note that there are still one issues with it: connections between copied chips are lost

**Comment by firecerne** [comment]
I'll work on "Edit Chips", but first i'll fix Import Chips.

**Comment by UkrainianBanderasCat** [comment]
What if we will add inventory like someone suggested earlier (there is also excellent design)

**Comment by XenBogo** [comment]
the ability to slide the chip bar without grabbing one

**Comment by XenBogo** [comment]
a chip for just placing some text on the area

**Comment by XenBogo** [comment]
resizable playspace. maybe even having multiple tabs to switch to in the same save
